### PR TITLE
Document `--allow-import` flag

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/docs/issues/916

### CODE REVIEW

File: runtime/fundamentals/security.md [hunk 1]
```diff
@@ -342,7 +342,44 @@ deno run -A script.s
...
+
+**Imports are only allowed using HTTPS**
+
+This allow list is applied by default for static imports, and by default if
+`--allow-import` flag is specified.
+
+```
+# allow dynamically importing code from `https://deno.land`
+$ deno run --allow-import main.ts
+```
```
Comment (bartlomieju) [inline runtime/fundamentals/security.md:377]: Add an actual example with code and terminal output when it's denied

File: runtime/fundamentals/security.md [hunk 2]
```diff
@@ -342,7 +342,44 @@ deno run -A script.s
...
+
+Note, that specifying an allow list for `--allow-import` will override the list
+of default hosts.
```
Comment (bartlomieju) [inline runtime/fundamentals/security.md:380]: @dsherret please verify that this is true

Comment (dsherret) [reply runtime/fundamentals/security.md:380]: Yes.

File: runtime/fundamentals/security.md [hunk 3]
```diff
@@ -342,7 +342,44 @@ deno run -A script.s
...
+```
+# allow importing code from `https://example.com`
+$ deno run --allow-import=example.com main.ts
+```
+
+By default Deno allows to import sources from following hosts:
+
+- `deno.land`
+- `esm.sh`
+- `jsr.io`
```
Comment (dsherret) [inline runtime/fundamentals/security.md:365]: also needs cdn.jsdelivr.net

File: runtime/fundamentals/security.md [hunk 4]
```diff
@@ -413,6 +413,44 @@ deno run --allow-ffi --deny-ffi=./libfoo.so script.ts
 deno run --deny-ffi script.ts
 ```
 
+### Importing from the Web
+
+Allow importing code from the Web. By default Deno limits hosts you can import
+code from. This is true for both static and dynamic imports.
+
+If you want to dynamic import code - either using `import()` or `new Worker()`
+API an additional permissions need to be granted. Importing from local file
```
Comment (crowlKats) [inline runtime/fundamentals/security.md:422]:
```suggestion
If you want to dynamically import code, either using the `import()` or the `new Worker()` APIs, additional permissions need to be granted. Importing from the local file
```

File: runtime/fundamentals/security.md [hunk 5]
```diff
@@ -413,6 +413,44 @@ deno run --allow-ffi --deny-ffi=./libfoo.so script.ts
...
+system [requires `--allow-read`](#file-system-read-access), but Deno also allows
+to import from `http:` and `https:` URLs. In such case you will need to specify
+an explicit `--allow-import` flag:
+
+```
+# allow importing code from `https://example.com`
+$ deno run --allow-import=example.com main.ts
+```
+
+By default Deno allows to import sources from following hosts:
```
Comment (crowlKats) [inline runtime/fundamentals/security.md:432]:
```suggestion
By default Deno allows importing sources from following hosts:
```

File: runtime/fundamentals/security.md [hunk 6]
```diff
@@ -413,6 +413,44 @@ deno run --allow-ffi --deny-ffi=./libfoo.so script.ts
...
+
+This allow list is applied by default for static imports, and by default to
+dynamic imports if the `--allow-import` flag is specified.
+
+```
+# allow dynamically importing code from `https://deno.land`
+$ deno run --allow-import main.ts
+```
+
+Note, that specifying an allow list for `--allow-import` will override the list
```
Comment (crowlKats) [inline runtime/fundamentals/security.md:451]:
```suggestion
Note that specifying an allow list for `--allow-import` will override the list
```

**Review by crowlKats — APPROVED** [review]

**Pull request merged**